### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by Sokartecnologi** [keras-team/keras-io#1362]
import numpy as np
from keras.models import Sequential
from keras.layers import Dense

# Create a dataset
X = np.array([[0, 0], [0, 1], [1, 0], [1, 1]])
y = np.array([0, 1, 1, 0])

# Define the model
model = Sequential()
model.add(Dense(4, input_dim=2, activation='relu'))
model.add(Dense(1, activation='sigmoid'))

# Compile the model
model.compile(loss='binary_crossentropy', optimizer='adam', metrics=['accuracy'])

# Train the model
model.fit(X, y, epochs=1000, verbose=0)

# Evaluate the model
loss, accuracy = model.evaluate(X, y, verbose=0)
print(f'Loss: {loss:.4f}, Accuracy: {accuracy:.4f}')

**Comment by sachinprasadhs** [comment]
Hi, Could you please explain the your issue. 
For guides and details related to Keras Sequential model, you can refer the document here https://keras.io/guides/sequential_model/
